bot/exts/moderation/defcon.py

```diff
@@ -124,134 +136,143 @@ async def defcon_group(self, ctx: Context) -> None:
-
-                delta = datetime.now() - enabled
-
-                self.bot.stats.timing("defcon.enabled", delta)
-        except Exception:
-            pass
-
-        error = None
-        try:
-            await self.bot.api_client.put(
-                'bot/bot-settings/defcon',
-                json={
-                    'name': 'defcon',
-                    'data': {
-                        # TODO: retrieve old days count
-                        'days': days,
-                        'enabled': action is not Action.DISABLED,
-                        'enable_date': datetime.now().isoformat()
-                    }
-                }
-            )
-        except Exception as err:
-            log.exception("Unable to update DEFCON settings.")
-            error = err
-        finally:
-            await ctx.send(self.build_defcon_msg(action, error))
-            await self.send_defcon_log(action, ctx.author, error)
-
-            self.bot.stats.gauge("defcon.threshold", days)
-
-    @defcon_group.command(name='enable', aliases=('on', 'e'), root_aliases=("defon",))
-    @has_any_role(*MODERATION_ROLES)
-    async def enable_command(self, ctx: Context) -> None:
-        """
-        Enable DEFCON mode. Useful in a pinch, but be sure you know what you're doing!
-
-        Currently, this just adds an account age requirement. Use !defcon days <int> to set how old an account must be,
-        in days.
-        """
-        self.enabled = True
-        await self._defcon_action(ctx, days=0, action=Action.ENABLED)
-        await self.update_channel_topic()
-
-    @defcon_group.command(name='disable', aliases=('off', 'd'), root_aliases=("defoff",))
-    @has_any_role(*MODERATION_ROLES)
-    async def disable_command(self, ctx: Context) -> None:
-        """Disable DEFCON mode. Useful in a pinch, but be sure you know what you're doing!"""
-        self.enabled = False
-        await self._defcon_action(ctx, days=0, action=Action.DISABLED)
-        await self.update_channel_topic()
-
-    @defcon_group.command(name='status', aliases=('s',))
+    @defcon_group.command(aliases=('s',))
     @has_any_role(*MODERATION_ROLES)
-    async def status_command(self, ctx: Context) -> None:
+    async def status(self, ctx: Context) -> None:
         """Check the current status of DEFCON mode."""
         embed = Embed(
             colour=Colour.blurple(), title="DEFCON Status",
-            description=f"**Enabled:** {self.enabled}\n"
-                        f"**Days:** {self.days.days}"
+            description=f"""
+                **Threshold:** {humanize_delta(self.threshold)}
+                **Expires in:** {humanize_delta(relativedelta(self.expiry, datetime.utcnow())) if self.expiry else "-"}
+                **Verification level:** {ctx.guild.verification_level.name}
+                """
         )
 
         await ctx.send(embed=embed)
 
-    @defcon_group.command(name='days')
+    @defcon_group.command(aliases=('t',))
```

Comment:
Maybe we could add the `days` alias, at least for a little bit.

Reply:
Not sure, feels like that'd be hiding the fact it's not limited to days anymore

Reply:
The thing is that if it is called with an integer it will assume that the value is in days. I don"t mind it staying like that tbh.